src/resource/file.ts

```diff
@@ -155,32 +173,48 @@ type SaveFileOptions = WriteFileOptions & {
  * ```{note} This function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Saves a file in a folder associated with the given URL. The final filename may or may
- * not be the given `slug`.
- *
- * If you know the [media type](https://developer.mozilla.org/en-US/docs/Glossary/MIME_type)
- * of the file you are attempting to save, then you should provide this in the
- * `options` parameter. For example, if you know your file is a JPEG image,
- * then you should provide the media type `image/jpeg`. If you don't know, or
- * don't provide a media type, a default type of `application/octet-stream` will
- * be applied (which indicates that the file should be regarded as pure binary
- * data).
- *
- * The Container at the given URL should already exist; if it does not, the returned Promise will
- * be rejected. You can initialise it first using [[createContainerAt]], or directly save the file
- * at the desired location using [[overwriteFile]].
- *
- * This function is primarily useful if the current user does not have access to change existing files in
- * a Container, but is allowed to add new files; in other words, they have Append, but not Write
- * access to a Container. This is useful in situations where someone wants to allow others to,
- * for example, send notifications to their Pod, but not to view or delete existing notifications.
- * You can pass a suggestion for the new Resource's name, but the server may decide to give it
- * another name — for example, if a Resource with that name already exists inside the given
- * Container.
- * If the user does have access to write directly to a given location, [[overwriteFile]]
- * will do the job just fine, and does not require the parent Container to exist in advance.
- *
- * @param folderUrl The URL of the folder where the new file is saved.
+ * Saves a file in an existing folder/Container associated with the given URL.
+ *
+ * For example:
+ *
+ * ```
+ * const savedFile = await saveFileInContainer(
+ *   "https://pod.example.com/some/existing/container/",
+ *   new Blob(["This is a plain piece of text"], { type: "plain/text" }),
+ *   { slug: "suggestedFileName.txt", contentType: "plain/text", fetch: fetch }
+ * );
+ * ```
+ *
+ * For additional example, see
+ * [Read/Write Files](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/read-write-files/#save-a-file-into-an-existing-container).
+ *
+ * In the `options` parameter,
+ *
+ * - You can suggest a file name in the `slug` field.  However, the Solid
+ *   Server may or may not use the suggested `slug` as the file name.
+ *
+ * - *Recommended.* You can specify the [media type](https://developer.mozilla.org/en-US/docs/Glossary/MIME_type)
```

Comment:
I'm not sure whether it's wrong, but having a sentence consisting of a single word looked weird to me, so thought I'd point that out :)

```suggestion
 * - *Recommended:* You can specify the [media type](https://developer.mozilla.org/en-US/docs/Glossary/MIME_type)
```